sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/SparkSQLEnv.scala

```diff
@@ -33,11 +33,11 @@ import org.apache.spark.util.Utils
 private[hive] object SparkSQLEnv extends Logging {
   logDebug("Initializing SparkSQLEnv")
 
-  var sqlContext: SQLContext = _
```

Comment:
I searched on GitHub and it doesn't seem like any notable projects are directly using `SparkSQLEnv.sqlContext`, and since this is an internal `var` of Spark, I have directly deleted it in this PR. 
If this variable is not suitable for direct deletion, please let me know, we can opt to mark it as `deprecated` in this pr.

Reply:
I don't know if we should delete it. cc @cloud-fan

Reply:
If there's a risk, I can revert this part and retain the `sqlContext` to avoid unnecessary trouble.

Reply:
change to mark `SparkSQLEnv.sqlContext` as `deprecated`